react/package.json

```diff
@@ -6,6 +6,7 @@
     "lint": "tsc --noEmit && tslint src/**/*.ts"
   },
   "dependencies": {
+    "@vtex/styleguide": "5.2.3",
```

Comment:
Let's use the loader from our `BuildStatus`. Render runtime should not depend on styleguide, specially from `npm`. It's possible to depend on styleguide as an app (in manifest.json) and get updates automatically, but unfortunately this would create a cyclic dependency.

Later we will have a mechanism to declare a `store/__loading` extension point and use it here, then `dreamstore-theme` could use a styleguide component as the default store loading.